src/Nri/Ui/FocusRing/V1.elm

```diff
@@ -53,9 +53,6 @@ forMouseUsers =
     , Css.Global.selector ":focus-within .checkbox-icon-container"
         [ Css.important (Css.boxShadow Css.none)
         ]
-    , Css.Global.selector ":focus-within .Nri-RadioButton-RadioButtonIcon"
-        [ Css.important (Css.boxShadow Css.none)
-        ]
```

Comment:
@tesk9 I thought really hard about if there was a good reason to have this selector here twice! Couldn't come up with any, pulled it, and it still works! 

Please let me know if this was indeed intentional!

Reply:
Oh nice catch! Just a mistake, thanks for fixing!